### DISCUSSION TITLE
join doesn't work on v5

### DISCUSSION THREAD

**Discussion by LordVersA** [adonisjs/core#1999] [Help]
there is nothing in document to help me in join
i use this , but result has no joined data:
```
const products = await Product.query().innerJoin(
      "categories",
      "category_id",
      "categories.id"
    );
```

**Comment by Imunal** [comment]
I'm having the same issue, any update on this?

**Comment by thetutlage** [comment]
"Doesn't work" is a broad statement and impossible to answer. Please provide the error you have received or the expected + actual result of the query.

Meanwhile you use the Knex docs for the Query builder methods https://knexjs.org/#Builder-innerJoin
